OAuth/Exception/StateRetrievalException.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the HWIOAuthBundle package.
+ *
+ * (c) Hardware.Info <opensource@hardware.info>
```

Comment:
We changed the brand name on July 15. I just pushed an update to master: https://github.com/hwi/HWIOAuthBundle/commit/7a13716e68faf185bd22aef20b37a523569f2ea3

Please rebase and run cs-fixer again :)